pallets/payment-streams/src/lib.rs

```diff
@@ -674,11 +689,15 @@ pub mod pallet {
             // Execute checks and logic, update storage
             let amount_charged = Self::do_charge_payment_streams(&provider_id, &user_account)?;
 
+            // Get the last tick to add it to the event
+            let last_tick_charged = Self::get_current_tick();
+
             // Emit the corresponding event (we always emit it even if the charged amount was 0)
             Self::deposit_event(Event::<T>::PaymentStreamCharged {
                 user_account,
                 provider_id: provider_id,
                 amount: amount_charged,
+                last_tick_charged,
```

Comment:
Same

Reply:
For example, here in `do_charge_payment_streams`, it would be the `last_chargeable_tick` variable used there.